lib/api/src/modules/release-notes.ts

```diff
@@ -0,0 +1,60 @@
+import { RELEASE_NOTES_DATA } from 'global';
+import memoize from 'memoizerific';
+
+import { ModuleFn, State } from '../index';
+
+export interface ReleaseNotes {
+  success?: boolean;
+  currentVersion?: string;
+  showOnFirstLaunch?: boolean;
+}
+
+const getReleaseNotesData = memoize(1)(
+  (): ReleaseNotes => {
+    try {
+      return { ...(JSON.parse(RELEASE_NOTES_DATA) || {}) };
+    } catch (e) {
+      return {};
+    }
+  }
+);
+
+export interface SubAPI {
+  releaseNotesVersion: () => string;
+  showReleaseNotesOnLaunch: () => Promise<boolean>;
+}
+
+export const init: ModuleFn = ({ fullAPI, store }) => {
+  let initStoreState: Promise<State[]>;
+  const releaseNotesData = getReleaseNotesData();
+
+  const api: SubAPI = {
+    releaseNotesVersion: () => releaseNotesData.currentVersion,
+    showReleaseNotesOnLaunch: async () => {
```

Comment:
This is sort of an unusual pattern. I feel like it's pretty confusing to have a `async` function called from a module's `initModule` given that we don't actually `await` initModule...